smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -363,8 +363,9 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
                     (await eventHubClient.GetRuntimeInformationAsync()).PartitionCount),
                 EventPosition.FromEnd());
 
+            // TODO: Don't rely on eventhub to verify module to module direct methods. The module sending the direct method should report success / failure directly.
```

Comment:
Suggest to put more positive statement
```suggestion
            // TODO: [Improvement] should verify test results without using event hub, which introduce latency.
```